apps/web-connect/index.html

```diff
@@ -14,6 +14,21 @@
     <meta http-equiv='expires' content='0'>
     <meta http-equiv='pragma' content='no-cache'>
 
+    <!--    OG Social Sharing Tags-->
+    <meta name="title" property="og:title" content="Xai Sentry Node">
+    <meta name="description" property="og:description" content="Xai Sentry Node Key Sale Page">
+    <meta name="image" property="og:image" content="https://xai-foundation.github.io/sentry-development/images/sentry-share.jpg">
```

Comment:
@CryptITAustria I need some guidance on where you feel this image file should live/be hosted and how it should be referenced from different domains.